services/common/src/components/forms/FormWrapper.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC, useEffect } from "react";
+import { Form } from "antd";
+import { compose } from "@reduxjs/toolkit";
+import { connect, useDispatch, useSelector } from "react-redux";
+import { reduxForm, submit, getFormSubmitErrors, InjectedFormProps, ConfigProps } from "redux-form";
+
+export interface IFormContext {
+  isEditMode: boolean;
+  isModal: boolean;
+}
+
+export const { Provider: FormProvider, Consumer: FormConsumer } = React.createContext<IFormContext>(
```

Comment:
This is really great!

I'm a bit concerned though about making sure everyone knows how to use it properly.  
I much prefer this pattern of wrapping a form with this over using redux form on the export the way we have been doing, but maybe we could add a couple of code comments here to explicitly state that this is the intention and how it will work?
Hopefully that will help ensure that it gets used!